spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/web/SpringDataWebProperties.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.data.web;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for Spring Data Web.
+ *
+ * @author Vedran Pavic
+ * @since 2.0.0
+ */
+@ConfigurationProperties("spring.data.web")
+public class SpringDataWebProperties {
+
+	private final Pageable pageable = new Pageable();
+
+	private final Sort sort = new Sort();
+
+	public Pageable getPageable() {
+		return this.pageable;
+	}
+
+	public Sort getSort() {
+		return this.sort;
+	}
+
+	/**
+	 * Pageable properties.
+	 */
+	public static class Pageable {
+
+		/**
+		 * Page index parameter name.
+		 */
+		private String pageParameter;
+
+		/**
+		 * Page size parameter name.
+		 */
+		private String sizeParameter;
+
+		/**
+		 * Default page size.
+		 */
+		private Integer defaultPageSize;
```

Comment:
OK, I see no harm to include the defaults from Spring Data - agreed both that they're not likely to change and documenting them indeed is more user friendly.